SparkPost/2web2ui
# AC-1161 - Launch new onboarding flow

AC-1161 - Launch new onboarding flow

Note - For checking out all guides use this url http://app.sparkpost.test:3100/

### What Changed
- Removed old onboarding code (NOT the page where you pick a plan)

> Delete
> - Tutorial
> - Onboarding - Sending domain
> - Onboarding - SmtpOrApi
> - Onboarding - API
> - Onboarding - SMTP
> - Steps

- Removed feature flag
- Set Onboarding guides from staging to production
- Reorder hierarchy of guides on production - onboarding should be top priority

Add:
- Link to dashboard with react-router state "fromOnboarding": true (to trigger modal)
- Add the condition to show the guide at bottom if all guides are completed.


### How To Test
 - Verify the above deletions and also verify whether related obsolete code is removed.
 - verify whether feature flag (onboarding_messaging) is removed.
 - Verify below behavior - 
1. On Creating a new account guide should be similar to this mock - https://sparkpost.invisionapp.com/share/DHULLYL45GB#/screens/390961092
2. Clicking on any further step should store the step and keep it at that on page refresh.

**Show Me SparkPost Checklist**
3. Clicking on "Send Test Email". Button directs to the Templates page / starts the guide https://sparkpost.invisionapp.com/share/DHULLYL45GB#/screens/390961096. 
4. Clicking on Explore Analytics to /reports/summary. Stores condition for check in account UI option on navigation. starts guide https://sparkpost.invisionapp.com/share/DHULLYL45GB#/screens/390961098.
5. Clicking on Invite a Collaborator redirects to /account/users
. "setup email sending now" links to the "Let's Code" checklist. 

**Let's Code Checklist**
6. verify functionality specified in AC https://jira.int.messagesystems.com/browse/AC-1088
7. verify functionality specified in AC https://jira.int.messagesystems.com/browse/AC-1089
8. verify functionality specified in AC https://jira.int.messagesystems.com/browse/AC-1168

### To Do
